tools/lib/wrap.sh

```diff
@@ -34,4 +34,10 @@ if [[ -z "${sdc_zone}" ]]; then
     echo "error: $(basename $0): unable to find a 'sdc' core zone on this node" >&2
     exit 1
 fi
+
+if [[ ! -x /zones/${sdc_zone}/root/opt/smartdc/sdc/bin/$(basename $0) ]]; then
```

Comment:
nit: is it worth it to store `/zones/${sdc_zone}/root/opt/smartdc/sdc/bin/$(basename $0)` in a variable to avoid duplication?